packages/replay-next/components/elements/ElementsListItem.tsx

```diff
@@ -165,6 +165,7 @@ export function ElementsListItem({
         style={
           {
             ...style,
+            width: undefined,
```

Comment:
This and the `<span>` wrapper change above will fix the horizontal overflow issue I noticed.